src/test.coffee

```diff
@@ -79,7 +69,7 @@ class Test
 
     # Body
     if @response.schema
-      schema = @parseSchema @response.schema
+      schema = @response.schema
```

Comment:
I think we should keep `test.schema` and `test.body` in same type. Now, `test.body` is dict and `test.schema` is a string. Do you have any better ideas?